### ISSUE TITLE
Rapidly Declining Server Population / Incentives

### ISSUE THREAD

**Issue by BGG100** [SmashMC-Development/Suggestions#1062]
***I want to preface this suggestion forum by stating that I have zero intent to come across as hostile or disrespectful. This is solely a suggestion post made out of both curiosity and concern for the near/far future of the network***

All players can agree that the network as a whole has seen a drastic decrease in server population in the past month, and many individuals are beginning to feel uninterested/unmotivated to continue playing on the network, due to the aforementioned population decrease among other issues currently prevalent. Every server is going to have minute glitches and bugs that will go undetected/unfixed for lengthened periods of times causing individuals to quit, however networks normally have incentives to persevere through said bugs and continue playing. This is were SmashMC lacks - there is a lack of incentive, especially for newer players, to see the server through its hardships and mishaps. 

For example, weekly/bi-weekly Pokemon Showdown OR in-game **tournaments** held at large creates optimal incentivization for players to play the server through hardship. In the current state of the server, this 'hardship' would be the plethora of bugs/missing plug-ins due to the recent update and reset to Pixelmon 9.0.0. At the moment. there is no reason for players to play through the difficulties created by said issues, causing newer players to seek out other networks with more frequent additions, events, and/or tournaments. **In saying this, I am NOT discrediting SmashMC events. It is evident that staff holds their own when it comes to in-game events as seen in the 'Events' section of the discord** The issue comes at the premise of the quality and incentives of the events. Anecdotally, I have seen a plethora of players join the server for the first time, only to leave a few minutes later and never return. The rapidly decreasing population of the network causes a repercussive cycle of continuous decrease, as the lifeless chats lead newer players to seek out more lively networks. 

As of late, each realm has been hosting about 10-20 players - most of which have paid in-game ranks, who have played on the network for a long time and have grown familiar and accustomed to it. This in itself is the most obvious sign of ineffective incentivization for newer players: and the continuous decrease in influx of new players is the result.

Once again, I am not bashing, hating, or disrespecting the server. With the past in mind, I do not wish to repeat any negativity conjured from my passion for the network. Since I first joined the network 3 years ago, the community of SmashMC has always been its strong point. Though I have seen SmashMC through it's ups and downs, this appears to be the smallest the community has ever dwindled to. I just hope there is possibility for this to be turned around sooner than later :)

**Comment by mnky-frt** [comment]
i agree tournaments would be a great thing that the server could offer

however i suggested this awhile a go and it was declined :(
https://github.com/SmashMC-Development/Suggestions/issues/569
